src/main/java/org/jenkinsci/account/BoardElection.java

```diff
@@ -63,7 +62,7 @@ private String messageDigest(String s) throws NoSuchAlgorithmException {
         MessageDigest md = MessageDigest.getInstance("MD5");
         md.update(s.getBytes());
         byte[] digest = md.digest();
-        return DatatypeConverter.printHexBinary(digest);
+        return s;
```

Comment:
Lazy workaround until https://github.com/jenkins-infra/account-app/pull/140 is merged